packages/firestore/test/integration/api/query.test.ts

```diff
@@ -140,8 +140,12 @@ apiDescribe('Queries', (persistence: boolean) => {
     });
   });
 
-  // Mirror queries are `Query`s mapped to the same `Target`, meaning
-  // they are supported by the same backend Query.
+  // Queries that mapped to the same target ID are referred to as
```

Comment:
s/mapped/map (I take credit for that one)

Also maybe start with "Two queries...".